inc/Integrations/SalesforceB2C/Auth/SalesforceB2CAuth.php

```diff
@@ -54,14 +56,15 @@ public static function get_token_using_client_credentials(
 		string $client_secret,
 		string $endpoint,
 		string $organization_id,
+		string $site_id,
 	): WP_Error|string {
 		$client_auth_url = sprintf( '%s/shopper/auth/v1/organizations/%s/oauth2/token', $endpoint, $organization_id );
 		$client_credentials = base64_encode( sprintf( '%s:%s', $client_id, $client_secret ) );
 
 		$client_auth_response = wp_remote_post($client_auth_url, [
 			'body' => [
 				'grant_type' => 'client_credentials',
-				'channel_id' => 'RefArch',
+				'channel_id' => $site_id,
```

Comment:
I noticed this was different than the `site_id` hardcoded elsewhere of `RefArchGlobal` so I've corrected this.